content/docs/06-integrations/00-antrea-cni.md

```diff
@@ -32,27 +32,35 @@ Antrea leverages [Open vSwitch](https://www.openvswitch.org/) to implement pod n
 # Prerequisites
 
 - Enable the integrated NodeIPAM controller in the Antrea manifest: ``NodeIPAM:enable``.
-- When deploying a cluster using Palette, use the ``podCIDR`` parameter in the Pack section of the Kubernetes manifest. The CIDER IP specified with ``podCIDR`` always takes precedence.
+<br />
+
+- When deploying a cluster using Palette, use the ``podCIDR`` parameter in the Pack section of the Kubernetes manifest. The classless inter-domain routing (CIDR) IP specified in the Kubernetes manifest always takes precedence.
+
+    <br />
 
-    When deploying a cluster using kubeadm to use Antrea CIDRs, specify the ``--pod-network-cidr <cidr>`` option and provide the IP address with the classless inter-domain routing (CIDR). For example: 
+- When deploying a cluster using ``kubeadm init`` to use Antrea CIDRs, you would do the following: 
+
+    - Specify the ``--pod-network-cidr <cidr>`` option and provide the IP address with the CIDR. For example: 
 
     <br />
 
     ``--pod-network-cidr=10.244.0.0/16``
 
     <br />
 
-    <WarningBox>
+    - Remove any values for the ``podCIDR`` and ``serviceCIDR`` parameters in the Kubernetes manifest. 
```

Comment:
As discussed, this is not applicable in case of `kubeadm init`. This can be removed from here.